pulsar-functions/python-examples/exclamation.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+
+from pulsarfunction import pulsar_function
+
+class Exclamation(pulsar_function.PulsarFunction):
```

Comment:
I think the package here looks a bit redundant.

This could be shortened into: 

```python
from pulsar import Function
class Exclamation(Function):
    def process(self, input, context):
        return input + '!'
```

or 

```python
impor pulsar
class Exclamation(pulsar.Function):
    def process(self, input, context):
        return input + '!'
```

We just need to figure out if there will be a clash with the `pulsar-client` Python library.

Reply:
/cc @srkukarni